### ISSUE TITLE
Great project! Is there any plan to add network support?

### ISSUE THREAD

**Issue by dean840513** [ktock/container2wasm#90]
Great project! Is there any plan to add network support?

**Comment by dean840513** [comment]
I apologize if I didn't make myself clear. What I meant was the situation when running the container in a browser.

**Comment by ktock** [comment]
Yes, there is. Wamr-based networking is on-going at https://github.com/ktock/container2wasm/pull/64 . We'll work on networking for browser and other wasi runtimes as well.

**Comment by ktock** [comment]
FYI: Initial support for networking is added to v0.5 .

Docs: https://github.com/ktock/container2wasm/tree/main/examples/networking

Network stack can run on browser (w/ limitations by CORS, etc.) or on host machine. Demo page (https://ktock.github.io/container2wasm-demo/) uses the former.

- [./examples/networking/fetch](https://github.com/ktock/container2wasm/blob/main/examples/networking/fetch): Running container on browser with on-browser network stack based on Fetch API.
- [./examples/networking/websocket](https://github.com/ktock/container2wasm/blob/main/examples/networking/websocket): Running container on browser with the network stack running on the host and accessible over WebSocket.
- [./examples/networking/wasi](https://github.com/ktock/container2wasm/blob/main/examples/networking/wasi): Running container on WASI runtimes (wasmtime and wazero), with network stack running on the host.

**Comment by ktock** [comment]
Closing this issue as the networking support is available since v0.5 . But please feel free to open this if you have suggestions for  networking support.
